.drone.yml

```diff
@@ -306,9 +306,10 @@ pipeline:
       status: success
 
   trigger-downstream:
-    image: 'gcr.io/eminent-nation-87317/vic-downstream-trigger:1.0'
+    image: 'gcr.io/eminent-nation-87317/vic-downstream-trigger:1.1'
```

Comment:
Does the build push these images as a pre-build step if there are Dockerfile changes? It doesn't does it....
Approved conditional on this having already been built and pushed.

Reply:
We don't currently do any automated builds of the container images we use for our build process. Once we have a dedicated repository for such tools, we can explore options for a CI/CD pipeline.